### PR TITLE
P4-2235 changes to the outputted filename and some minor refactorings.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
Changes include:

Updates to the outputted spreadsheet.

Cleaning up some of the unit tests.  Getting rid of unnecessary code.

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/TestConfig.kt [hunk 1]
```diff
@@ -24,18 +20,18 @@ class TestConfig {
     fun clock(): Clock = Clock.fixed(Instant.now(), ZoneId.systemDefault())
 
     @Bean
-    fun locationsResourceProvider(@Value("\${import-files.locations}") locationsFile: String): Schedule34LocationsProvider {
```
Comment (matt-crowson-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/TestConfig.kt:27]: Don't need these properties in here really so just binned them.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/output/PricesSpreadsheetGeneratorTest.kt [hunk 2]
```diff
@@ -16,9 +14,7 @@ import uk.gov.justice.digital.hmpps.pecs.jpc.reporting.FilterParams
 import java.time.Clock
 import java.time.LocalDate
 
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@ActiveProfiles("test")
-@ContextConfiguration(classes = [TestConfig::class])
+@SpringJUnitConfig(TestConfig::class)
```
Comment (matt-crowson-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/output/PricesSpreadsheetGeneratorTest.kt:17]: 3 annotations down to 1, much cleaner.

File: src/test/resources/application-test.yml [hunk 3]
```diff
@@ -47,13 +47,6 @@ management:
 logging:
   level:
     ROOT: INFO
-import-files:
```
Comment (matt-crowson-moj) [inline src/test/resources/application-test.yml:50]: Don't need these as moved directly to the TestConfig.kt class.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/ImportController.kt [hunk 4]
```diff
@@ -26,10 +29,13 @@ class ImportController(private val importService: ImportService) {
...
 
         return importService.spreadsheet(supplier, movesFrom, movesTo, reportsTo)?.let { file ->
+            val uploadDateTime = LocalDateTime.now(clock).format(DateTimeFormatter.ofPattern("YYYY-MM-DD_HH_MM"))
+            val filename = "Journey_Variable_Payment_Output_${supplier}_${uploadDateTime}.xlsx"
             val mediaType: MediaType = MediaType.parseMediaType("application/vnd.ms-excel")
             val resource = InputStreamResource(FileInputStream(file))
+
             ResponseEntity.ok()
-                    .header(HttpHeaders.CONTENT_DISPOSITION, "attachment;filename=$supplier-prices.xlsx")
+                    .header(HttpHeaders.CONTENT_DISPOSITION, "attachment;filename=$filename")
```
Comment (jcavell) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/ImportController.kt:38]: Be nice to have a test

**Review by jcavell — APPROVED** [review]

**Pull request merged**